pkg/graph/schema.resolvers.go

```diff
@@ -1849,10 +1849,14 @@ func (r *systemIntakeResolver) Requester(ctx context.Context, obj *models.System
 		return nil, err
 	}
 
+	// if we can't find the user and there was no error (shouldn't happen normally), omit the email
+	// user is a pointer, so we want to avoid a dereference below with this check
 	if user == nil {
-		return nil, nil
+		return requesterWithoutEmail, nil
```

Comment:
@samoddball I think this is more in-line with current functionality, right?

Reply:
looks like it! based on the naming of `requesterWithoutEmail`, it looks like we are aware that a user can exist without an email and would act accordingly in the caller? otherwise could just return err and end processing 🤷, but i see the same thing is done earlier in the func, so it is up to you!